core/modules/system/css/system.css

```diff
@@ -31,15 +31,16 @@
 }
 /* Animated throbber */
 .js input.form-autocomplete {
-  background-image: url(../../../misc/throbber.gif);
-  background-position: 100% 2px; /* LTR */
+  background-image: url(../../misc/throbber-inactive.png);
```

Comment:
Looks like this would cause a broken image. We need 3 `../` parent directories up.